### ISSUE TITLE
Espial subpath?

### ISSUE THREAD

**Issue by risturiz** [jonschoning/espial#58]
Hi, i see there are some variables on settings.yml but i dont know if i can configure espial to run on subpath https://www.domain.com/espial ?

Thanks for any advice :-)

**Comment by jonschoning** [comment]
The easiest way will probably be using some sort of Reverse Proxy.  [nginx](https://github.com/jonschoning/espial/issues/6) is well-known, but can be complex to setup.
You could also try something simple like [Caddy](https://caddyserver.com/docs/quick-starts/reverse-proxy#caddyfile) which accepts different [subpath matchers](https://caddyserver.com/docs/caddyfile/matchers#examples)

**Comment by risturiz** [comment]
Yes, im using NPM... For now i have https://espial.domain.com working with docker... I need to figure out how to rewrite url like /auth/login & /u:user ( so i can have www.domain.com/espial/auth/login )... Thanks!

**Comment by jonschoning** [comment]
corrected behavior on [Release V0.0.38](https://github.com/jonschoning/espial/releases/tag/v0.0.38); see https://github.com/jonschoning/espial#running-on-a-subpath
